noxfile.py

```diff
@@ -138,7 +137,7 @@ def scripts(session):
         "--cov=scripts",
         "--cov-config=.coveragerc",
         "--cov-report=",
-        "--cov-fail-under=91",
+        "--cov-fail-under=59",
```

Comment:
Why are we decreasing the coverage here? I would keep it as high as we can, so we can be notified as early as possible of any breakages.

Reply:
Reverted